examples/riot-kitchen-sink/src/stories/compileNow.js

```diff
@@ -0,0 +1,13 @@
+import compiler from 'riot-compiler';
```

Comment:
Is this helper always exist in users code? Or also can be hidden in `@storybook/riot`?

Reply:
done